master/internal/api_auth.go

```diff
@@ -73,7 +73,7 @@ func (a *apiServer) Login(
 	if !userModel.Active {
 		return nil, grpcutil.ErrNotActive
 	}
-	token, err := a.m.db.StartUserSession(userModel)
+	token, err := user.StartUserSession(context.TODO(), userModel)
```

Comment:
go ahead and use the available context since we can now here and everywhere else it is easy. awesome.